codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -108,8 +108,15 @@ public void channelHandlerContext(ChannelHandlerContext ctx) {
         this.ctx = ctx;
     }
 
+    /**
+     * {@inheritDoc}
+     * <p>
+     * This method must be called from the same event loop context as the {@link ChannelHandlerContext}
```

Comment:
@nmittler - I'm not against it but I'm not crazy about using these annotations either.  For example we don't really use `@ThreadSafe` or `@NotThreadSafe` now, and so if there is no annotation...what should the user assume about this?  I guess the point is we don't necessarily want them to assume anything just because the annotation is not present, but by explicitly using `@NotThreadSafe` they may think if this is not present that the method is thread safe (despite the javadoc on the annotation saying nothing should be assumed).